rfd/0074-sftp-support.md

```diff
@@ -0,0 +1,56 @@
+---
+authors: Andrew LeFevre (andrew.lefevre@goteleport.com)
+state: draft
+---
+
+# RFD 74 - SFTP Support
+
+## What
+
+Add SFTP support to `tsh` and Teleport Node and Proxy services.
```

Comment:
Is SFTP intended to replace SCP, or will we support both? Maybe be more explicit here.

Reply:
I'm assuming we want to support both at least for awhile, but not sure if we have any plans of deprecating scp in the future